examples/b2c/checkout/payment-gateway-integration/StripeClientSide/classes/QueryUtils.cls

```diff
@@ -0,0 +1,292 @@
+// =============================================================================================
```

Comment:
We had forceDataSoql method from sfdx utils class (you can look it for reference). Does this class do something particularly different?

Reply:
This is an apex class used by the sample payment reference implementation that we added.  It has nothing to do with plugin/commands.  It gets deployed to the org when you run `sfdx commerce:payments:quickstart:setup --payment-adapter StripeClientSide`